Light.json

```diff
@@ -6138,6 +6140,7 @@
         "type": "Character",
         "subType": "Alien",
         "uniqueness": "*",
+        "destiny": "π",
```

Comment:
I'm not sure about this one, should we have value be 3.14 instead?  That was if you searched for destiny greater than 3 it should show up without special handling.

Reply:
It indeed depends on how downstream clients consume this information, but I do feel that there should be a destiny field (this one or a new one) that simply shows the destiny as it appears on the card.

I noticed that EG-4/EG-6 uses `"*"`, so someone is broken already if they're treating these as numbers. Hehehe

Reply:
Yeah, you're right lets use the pi symbol.  Comlink should be fine with these changes.

Reply:
I used to stress over ensuring destinies were numbers that could be parsed, but I eventually realized that the list of exceptions is so short that any system needing raw values could easily handle the tiny list of odd destinies.